app/scripts/run-apis.sh

```diff
@@ -15,10 +15,22 @@ docker run --rm \
     $registry/montagu-cert-tool:$cert_tool_version \
     gen-keypair /workspace
 
+docker volume prune -f
+
 # Run the APIs and database
 docker-compose pull
 docker-compose --project-name montagu up -d
 
+# Generate report test data
+docker pull $registry/orderly:$orderly_version
+docker run --rm \
+    --entrypoint create_orderly_demo.sh \
+    -v montagu_orderly_volume:/orderly \
+    $registry/orderly:$orderly_version \
+    /orderly
+
+docker exec montagu_reporting_api_1 sh -c 'cp /orderly/demo/. /orderly/ -r'
+
```

Comment:
this is a workaround as the orderly task generates the 'demo'  folder we want as a subfolder of the orderly directory